sigmf/utils.py

```diff
@@ -18,7 +18,7 @@
 
 def get_sigmf_iso8601_datetime_now() -> str:
     """Get current UTC time as iso8601 string"""
-    return datetime.isoformat(datetime.utcnow()) + "Z"
+    return datetime.isoformat(datetime.now(UTC)) + "Z"
```

Comment:
Not sure why but ok.

Reply:
Looks like this part is failing in tests. Looks like this feature was added in `3.11`, so if you revert this part it will work in `3.6`.

Reply:
Hi @Teque5,

> Nice PR; used SigMFFileError and even included a test for the issue.

Thanks! 

we can drop the `datetime` change, it's pretty irrelevant. I was just getting a warning and thought I'd address it while I was at it.